app/components/contact/contact-item/contact-item-modal.js

```diff
@@ -0,0 +1,50 @@
+import Component from '@glimmer/component';
+import { inject as service } from '@ember/service';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+import { isBlank } from '@ember/utils';
+
+export default class ContactContactItemModalComponent extends Component {
```

Comment:
Ik bedoelde hier een subfolder `contact-item` direct onder `components` en niet in `components/contact`. Zelfde voor de table component.

In lijn met de componenten onder `contact` en `press-release` zou ik die dan ook gewoon `modal.js` en `table.js` noemen.